test/src/helper/mock/blockSyncMessage.ts

```diff
@@ -64,14 +63,10 @@ export class BlockSyncMessage {
         if (msgId === MessageType.MESSAGE_ID_STATUS) {
             Emitter.emit("status");
             const msg = decodedmsg[1];
-            const totalScore = new U256(
-                parseInt(msg[0].toString("hex"), 16) || 0
-            );
-            const bestHash = new H256(msg[1].toString("hex"));
-            const genesisHash = new H256(msg[2].toString("hex"));
+            const bestHash = new H256(msg[0].toString("hex"));
+            const genesisHash = new H256(msg[1].toString("hex"));
```

Comment:
I tried to change this code as follows.
```
const bestHash = new H256(msg[1].toString("hex"));
const genesisHash = new H256(msg[2].toString("hex"));
```
Then, an unknown time out error occurs, such as [the first error](https://github.com/CodeChain-io/foundry/pull/236#issuecomment-595698293).

Reply:
It seems that you need to know how the mock is working.

Source code in the test/src/helper/mock directory communicates with Foundry using the P2P messages. Your PR changed the mock code on how to decode the Status message. Since the format of the Status message is not changed, your code does not work.

Let me show the definition of the Status message at the current master branch: https://github.com/CodeChain-io/foundry/blob/master/sync/src/block/message/mod.rs#L63

IMO, you should rename the totalScore in this file to "seq". You should not remove the field.